### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by thecodechaser**
## This branch includes the following features:
- Written unit tests for the classes: 
  - Book
  - Person
  - Student
  - Teacher
  - Classroom
  - Corrector
  - Rental

### CODE REVIEW

File: spec/person_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    it 'should validate the name of the person' do
+      expect(person.name).to eq name
+    end
+
+    it 'should validate parent permission for parent' do
+      expect(person.parent_permission).to be parent_permission
+    end
+  end
+end
```
Comment (TedLivist) [inline spec/person_spec.rb]: - [ ] I would recommend you add **negative** and **positive** case scenario tests for the `can_use_services?` method in the Person class.

- [ ] Kindly add unit tests for the `validate_name` method.

- [ ] Kindly add unit tests for the `add_rental` method in the Person class.

File: spec/student_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    it 'should validate the name of the student' do
+      expect(student.name).to eq name
+    end
+
+    it 'should return parent permission for student' do
+      expect(student.parent_permission).to be parent_permission
+    end
+  end
+end
```
Comment (TedLivist) [inline spec/student_spec.rb]: - [ ] Kindly add a test for the `play_hooky` method in the Student class.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @thecodechaser and @akucintavalent,

Good job so far!
✔️ Tests are passing
✔️ Tests are wrapped in `describe` and `it` blocks
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### 🛑  Look here too 🛑 
- [ ] **In your README, kindly add instructions on how to run the tests**

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @thecodechaser, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**